src/auth/effects/auth.effects.ts

```diff
@@ -71,41 +72,44 @@ export class AuthEffects {
   /** TODO: Autologin after register, replace form.email with form.username
    * (waiting for a decision https://github.com/EyeSeeTea/FIRE-WiFiCalling/issues/37) */
 
-  @Effect({dispatch: false})
+  @Effect({ dispatch: false })
   registerSuccess$ = this.actions$
     .ofType(Auth.REGISTER_SUCCESS)
     .do(() => {
 
-      const alert = this.alerts.create({
-        title: 'Success',
-        message: 'Your account request will be reviewed by the admin.',
-        buttons: ['OK']
-      });
-      alert.present();
+      this.modalCtrl.create(DialogComponent,
+        {
+          title: 'Success',
```

Comment:
I am passing these values to the [ModalController](https://ionicframework.com/docs/api/components/modal/ModalController/) which loads our dialog template.
In this effect for instance only success message will be shown.
I will improve the error messages and convert these to work with the translation pipe.